mixins.scss

```diff
@@ -40,197 +43,213 @@
 	}
 }
 
-/**
- * @description
- * Generates `appearance` for a given element
- *
- * @author romamatusevich
- *
- * @link MDN https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance
- * @link css-tricks http://css-tricks.com/almanac/properties/a/appearance/
- * @link spec http://www.w3.org/TR/2004/CR-css3-ui-20040511/#appearance
- *
- * @param value
- * @returns
- *   -webkit-appearance: <value>;
- *      -moz-appearance: <value>;
- *           appearance: <value>;
- *
- * @example
- *   .selector {
- *     @include x-appearance(button);
- *   }
- */
+///
+/// Generates appearance for a given element
+///
+/// @author romamatusevich
+///
+/// @param {String} $value
+///
+/// @output
+/// ```css
+///   -webkit-appearance: <value>;
+///      -moz-appearance: <value>;
+///           appearance: <value>;
+/// ```
+///
+/// @example
+///   .selector {
+///     @include x-appearance(button);
+///   }
+///
 @mixin x-appearance ($value) {
 	-webkit-appearance: $value;
 	   -moz-appearance: $value;
 	        appearance: $value;
 }
 
-/**
- * @description
- * Generates `background-size` output for a given element.
- *
- * @author drublic
- *
- * @link caniuse http://caniuse.com/background-img-opts
- * @link spec http://www.w3.org/TR/css3-background/
- *
- * @param value
- * @returns
- *   -webkit-background-size: $args;
- *           background-size: $args;
- *
- * @example
- *   .selector {
- *     @include x-background-size(100% auto);
- *   }
- */
+///
+/// Generates `background-size` output for a given element.
+///
+/// @author drublic
+///
+/// @link http://caniuse.com/background-img-opts caniuse
+/// @link http://www.w3.org/TR/css3-background/ spec
+///
+/// @param {String} $value
+///
+/// @output
+/// ```css
+///   -webkit-background-size: $args;
+///           background-size: $args;
+/// ```
+///
+/// @example
+///   .selector {
+///     @include x-background-size(100% auto);
+///   }
+///
 @mixin x-background-size ($values...) {
     -webkit-background-size: $values; // For Android 2.x
             background-size: $values;
 }
 
-/**
- * @description
- * Generates cross-browser-compatible `border-radius` for a given element
- *
- * @author drublic
- *
- * @link caniuse http://caniuse.com/border-radius
- * @link spec http://www.w3.org/TR/css3-background/#corners
- *
- * @param values
- * @returns
- *   -webkit-border-radius: <values>;
- *           border-radius: <values>;
- *
- * @example
- *   .selector {
- *     @include x-border-radius(20px 10px);
- *   }
- */
+///
+/// Generates cross-browser-compatible `border-radius` for a given element
+///
+/// @author drublic
+///
+/// @link http://caniuse.com/border-radius caniuse
+/// @link http://www.w3.org/TR/css3-background/#corners spec
+///
+/// @param {String} $values
```

Comment:
I'd argue the value of the `border-radius` property is either a number or a list.